src/lib.rs

```diff
@@ -56,8 +57,6 @@ use std::fmt;
 use std::error;
 use libc::PATH_MAX;
 
-pub type Result<T> = result::Result<T, Error>;
```

Comment:
Oh, it should probably be moved back. Mostly it's there because that's where #230 had it.